src/styles/index.scss

```diff
@@ -1,3 +1,54 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
```

Comment:
You should move the font-family and font-size to a separate file called `typography.scss` in the `utils` folder. It's a good practice to separate the typography styles into a separate file. It makes the code more organized and easier to maintain.